checker/checker_test.go

```diff
@@ -1,10 +1,11 @@
-package main
+package checker
```

Comment:
Can this be renamed to the test package? Might not be able to if it checks private functions but I always try to if I can.
```suggestion
package checker_test
```
NB. same for [deployment/deployment_test.go](https://github.com/ONSdigital/dis-web-mount-check/pull/5/files#diff-eb2d2773e5ea56b06ee655e7e8088f07236d2066882d3bf8da096e37af0a58a3R1) too.

Reply:
done